pkg/agent/proxy/proxier.go

```diff
@@ -554,13 +551,16 @@ func NewProxier(
 }
 
 func NewDualStackProxier(
-	hostname string, informerFactory informers.SharedInformerFactory, ofClient openflow.Client) k8sproxy.Provider {
+	hostname string,
+	informerFactory informers.SharedInformerFactory,
+	ofClient types.ServiceClient,
+	enableEndpointSlice bool) k8sproxy.Provider {
```

Comment:
same as for comment in `cmd/antrea-agent/agent.go `

Reply:
Done.